cmdutil/signals/config.go

```diff
@@ -0,0 +1,6 @@
+package signals
+
+// Config describes the configurable parameters for Signals server.
+type Config struct {
+	ServerCloseWaitTime int `env:"SERVER_CLOSE_WAIT_TIME,default=0"`
```

Comment:
Maybe we should namespace the environment variable and make its purpose more explicit, for example, `SIGNALS_SERVER_STOP_DELAY`.

The type should be `time.Duration` and the default `0s`.